client/lib/cart/store/index.js

```diff
@@ -241,6 +241,7 @@ function createListener( store, selector, callback ) {
 
 // Subscribe to the Redux store to get updates about the selected site
 getReduxStore().then( store => {
-	CartStore.setSelectedSiteId( getSelectedSiteId( store.getState() ) );
+	const selectedSiteId = getSelectedSiteId( store.getState() );
+	selectedSiteId && CartStore.setSelectedSiteId( selectedSiteId );
```

Comment:
What does this line of code do?

Reply:
@dzver Answered in this [comment](https://github.com/Automattic/wp-calypso/pull/33749#discussion_r295362510).